README.md

```diff
@@ -1,63 +1,41 @@
+conda config --append channels conda-forge --append channels bioconda
+conda install --file requirements.txt
 ```
 
 ## Usage
 
 ### Explaining a gene's responsiveness to a TF perturbation
 
-`TFPRExplainer` predicts which gene would response to a TF perturbation and explains the extend to which each feature contributes to the prediction of responsiveness. Use the following example code to load feature matrix, cross validate response predictions, and analyze contributions of relevant genomic features.
+`TFPRExplainer` predicts which gene would response to a TF perturbation and explains the extend to which each feature contributes to the prediction of responsiveness. Use the following example code to load feature matrix, cross validate response predictions across TFs, and analyze contributions of relevant genomic features.
 
 It takes two forms of parameters:
 
-- *Command line arguments* define the perturbed TF, the collection of genomic features, and directory paths for input and output data. Use `--help` for details.
+- *Command line arguments* define the perturbed TF, the collection of genomic features, and directory paths for input and output data. Use `-h` for details.
 - *Configuration parameters* define the boundary of each gene's regulatory DNA, and instructions on processing feature matrix and response label. Reference `config.ini` for default parameters.
 
 For yeast genome, run
 
 ```
 $ python3 CODE/explain_yeast_resps.py \
-    -i YLR451W \
+    -i <tf1 tf2 ... tfn> \
     -f tf_binding histone_modifications chromatin_accessibility dna_sequence_nt_freq gene_expression gene_variation \
-    -x RESOURCES/h5_data/yeast_dna_cc_hm_atac_tss1000to500b_expr_var.h5 \
-    -y RESOURCES/Yeast_ZEV_IDEA/ZEV_15min_shrunkenData.csv \
-    -o OUTPUT/Yeast_CallingCards_ZEV/all_feats/
+    -x <feature matrix (.h5)> \
+    -y <tf-pert response data (.csv)> \
+    -o <output directory>
 ```
 
-For human genome, run
-
-```
-$ python3 CODE/explain_human_resps.py \
-    -i ENSG00000001167 \
-    -f tf_binding histone_modifications chromatin_accessibility dna_sequence_nt_freq gene_expression gene_variation \
-    -x RESOURCES/h5_data/human_encode_enhan_alltss_2kbto2kb_promo.h5 \
-    -y RESOURCES/HumanK562_TFPert/K562_pertResp_DESeq2_long.csv \
-    -o OUTPUT/Human_ChIPseq_TFpert//all_feats/
-```
-
-### Explaining a gene's frequency of response across perturbations
-
-```
-$ python3 CODE/explain_yeast_resps.py \
-    --is_regressor \
-    -i freq \
-    -f histone_modifications chromatin_accessibility dna_sequence_nt_freq gene_expression gene_variation \
-    -x RESOURCES/h5_data/yeast_dna_cc_hm_atac_tss1000to500b_expr_var.h5 \
-    -y RESOURCES/Yeast_ZEV_IDEA/ZEV_15min_shrunkenData_deFreq.csv \
-    -o OUTPUT/Yeast_ZEV_DE_freq/tf_indep_feats/xgb/
-```
+For human genome, use `CODE/explain_human_resps.py` with the above set of arguments.
```

Comment:
We could add a description of the flag for enabling SHAP value calculation below.

Reply:
Updated.